Studentmediene/kapina-frontend
# Feat/next server render

How to test:
* `npm install`
* `PRODUCTION_API=true npm start`
=> Browse around, check that everything works as it should. There is a known bug where css/scss won't get loaded while browsing, but this only affects the dev environment.

To test the production environment:
* `npm run build`
* `PRODUCTION_API=true npm run start:prod`

Check that everything works in the same way here aswell. This is exactly how it will look in production, so make sure that the bug with css/scss is not present.

Changelog:
* NextJS is now used to server-side render page on first load
* Support for web crawlers such as Facebook, Google etc
* Cached Post and Show component
* Removed LOTS of old comments and unused actions/reducers
* Remove most custom webpack config. NextJS fixes that for us ✌️ 
* Update/remove old packages in package.json
* convert all CSS files to SASS
* update linter rules
* Player component now only loads on client
